deliverables/Deliverable_6.md

```diff
@@ -0,0 +1,46 @@
+Group 4: **TimeSculptor**
+
+**Date:** November 19, 2023
+
+**Group Members:**
+
+- Gabriel Valentin
+- Izaac Molina
+- Joey Banazsak
+- Kimberly Allison
+- Nicholas Wiley
+- Noah Schwartz
+
+# Introduction
+ TimeSculptor is a scheduling app that aims to provide a seemless and user friendly experience, with a particular emphasis on being accessible for people with learning disabilities. It provides an accessible interface that adheres to guidelines outlined in [Pavlov (2014)](https://www.scirp.org/html/7-9301792_43152.htm) and [Moreno et al. (2023)](https://link.springer.com/article/10.1007/s10209-023-00986-z). The primary expected consumer segment individuals who possess learning disabilities, but this ultimately makes the app accessible for everyone. Upon logging in, you are then given the options to create events or edit your schedule's current events. You can utilize color coding, and assigning different symbols to your events. Additionally, there are three viewing modes: list, monthly, and daily.
+
+[TimeSculptor](https://github.com/nickw409/TimeSculptor)
+
+# Implemented Requirements
+
+
+
+# Tests
+
+For the testing of our program we decided to use jest as our automated testing software. 
+
+link for testing: [Here](https://github.com/nickw409/TimeSculptor/tree/main/TimeSculptor/src/components/__tests__)
+
```

Comment:
an example of a test case is the [calendar.jsx](https://github.com/nickw409/TimeSculptor/tree/main/TimeSculptor/src/components/calendar.jsx) file, in our tests, the first test in the [calendar.test.js](https://github.com/nickw409/TimeSculptor/tree/main/TimeSculptor/src/components/__tests__/calendar.test.js) file is seeing that the correct month is displayed based on a set of mock events and date.